torch/distributions/transforms.py

```diff
@@ -298,9 +301,60 @@ def log_abs_det_jacobian(self, x, y):
         return x
 
 
-class SigmoidTransform(Transform):
+class PowerTransform(Transform):
+    r"""
+    Transform via the mapping :math:`y = x^{\text{exponent}}`.
     """
-    Transform via the mapping `y = sigmoid(x)` and `x = logit(y)`.
+    def __init__(self, exponent):
+        self.exponent = exponent
+
+    # Currently only implementing this for instances of numbers.Number
+    @constraints.dependent_property
+    def domain(self):
+        frac = Fraction(Decimal(str(self.exponent)))
+        if frac.denominator % 2 == 1:
+            return constraints.real
+        else:
+            return constraints.positive
+
+    @constraints.dependent_property
+    def codomain(self):
+        frac = Fraction(Decimal(str(self.exponent)))
+        if frac.numerator % 2 == 1:
+            return constraints.real
+        else:
+            return constraints.positive
+
+    @constraints.dependent_property
+    def bijective(self):
+        frac = Fraction(Decimal(str(self.exponent)))
+        if frac.numerator % 2 == 1 and frac.denominator % 2 == 1:
+            return True
+        else:
+            return False
+
+    def __eq__(self, other):
+        if not isinstance(other, PowerTransform):
+            return False
+        else:
+            if other.exponent == self.exponent:
+                return True
+            else:
+                return False
+
+    def _call(self, x):
+        return x.pow(self.exponent)
+
+    def _inverse(self, y):
```

Comment:
This should always return `x.pow(1 / self.exponent)`. In the non-bijective case we still want to return a pseudoinverse, as do other transforms.